aireview/src/aireview/infrastructure/dust_client.py

```diff
@@ -45,7 +42,6 @@ def create_conversation(self, prompt: str):
             "blocking": True
         }
```

Comment:
**style**: Consider adding error handling with specific exception types rather than just raise_for_status() to provide more context about API failures.